components/x-teaser-list/src/TeaserList.scss

```diff
@@ -0,0 +1,34 @@
+@import 'o-colors/main';
+
+:global {
+	@import "~@financial-times/x-article-save-button/dist/ArticleSaveButton";
+}
```

Comment:
Is there a risk this chunk get duplicated if somebody includes the save button component directly?

(I'm not suggesting any fix for this problem, but it's probably worth a comment)